clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/sagas/SagaAtomicOperationBridge.java

```diff
@@ -55,18 +56,23 @@ public SagaAtomicOperationBridge(SagaService sagaService, String sagaId) {
 
     applyCommand.sagaFlow.injectFirst(SnapshotAtomicOperationInput.class);
 
-    return sagaService.applyBlocking(
-        sagaName,
-        sagaId,
-        applyCommand.sagaFlow,
-        SnapshotAtomicOperationInput.SnapshotAtomicOperationInputCommand.builder()
-            .cloudProvider(sagaContext.getCloudProvider())
-            .descriptionName(sagaContext.getDescriptionName())
-            .descriptionInput(sagaContext.getOriginalInput())
-            .description(applyCommand.inputDescription)
-            .priorOutputs(applyCommand.priorOutputs)
-            .nextCommand(applyCommand.initialCommand)
-            .build());
+    try {
+      MDC.put("X-SAGA", sagaName + "/" + sagaId);
```

Comment:
Minor -- In case we have sagas in other services do we want to identify the saga by the service (clouddriver in this case)?

Reply:
We have enough additional context when we log to be able to differentiate between services, I'm not sure we need more than that in this case.